CheatSheets/angular-cheatsheet.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Angular CheatSheet
+description: Angular cheatsheet contains basics of angular binding & angular CLI.
+created: 2022-10-03
+updated: 2022-10-03
+---
```

Comment:
Looks to me good.

Reply:
@crescentpartha Please assign me. I will do that changes.

Thanks!

Reply:
@crescentpartha I have done with Tabular format.

https://github.com/crescentpartha/CheatSheets-for-Developers/pull/165

Please Review above PR and Let me know if any changes required.

Thanks!

Reply:
Thanks @nikhilbuts Just asking will i get my PR accepted with the help of your contribution?